spec/requests/registrations_spec.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Registration', type: :request do
+  describe 'Email registration' do
+    let(:signup_url) { '/auth' }
+    let(:signup_params) do
+      {
+        email: Faker::Internet.email,
+        password: Faker::Internet.password
+      }
```

Comment:
I think the card is requesting for the user to provide a name, email, gender, password, and password confirmation.

Reply:
Updated!

Reply:
I think name is missing.